experiments/pages/lightningModalLauncher/lightningModalLauncher.xml

```diff
@@ -0,0 +1,202 @@
+							<styleVariantConditions/>
+						</skuid__button>
+					</buttons>
+				</skuid__buttonGroup>
+			</groups>
+			<styles>
+				<spacing bottom="2"/>
+			</styles>
+			<renderConditions logictype="and"/>
+			<styleVariantConditions/>
+		</skuid__buttonSet>
+	</components>
+	<resources>
+		<labels/>
+		<javascript>
+			<jsitem location="inline" name="EventHotFix" cachelocation="false" url="">((skuid) =&gt; {
+    window.skuid.model = {};
+	window.skuid.component = {};
+    window.skuid.model.Model = Object;
+    window.skuid.component.Component = Object;
+    console.log(8);
+})(skuid)</jsitem>
+			<jsitem location="inlinesnippet" name="callFlow" cachelocation="false">const params = arguments[0],
+    eventName = "showModalFlow",
+    flowName = params.$Input.flowName,
+    model = skuid.$M(params.$Input.model);
+console.log(this, skuid, params);
+
+// See if there is a global application event with this name
+let auraEvt = window.$A.get("e." + eventName),
+    skuidAuraEvt = window.$A.get("e.skuid:event"),
+    paramsObject = {
+        flowName: flowName,
+        params: model.getRows()
+    };
+
+// If there is a matching global application event with this name,
+// then fire that event, with each Skuid Event parameter
+// being mapped directly to that Event parameter.
+if (auraEvt) {
+    if (paramsObject) {
+        auraEvt.setParams(paramsObject);
+    }
+    auraEvt.fire();
+}
+
+skuidAuraEvt.setParams({
+    "name": eventName,
+    "source": "skuid",
+});
+
+// ALSO fire a generic "Skuid Event",
+// where all of the parameters are serialized to JSON
+if (paramsObject) {
+    skuidAuraEvt.setParam("data", paramsObject);
+}
+
+skuidAuraEvt.fire();</jsitem>
+			<jsitem location="inlinesnippet" name="showVariable" cachelocation="false">const params = arguments[0];
+const eventName = "showModalFlow";
+console.log(this, skuid, params);
+
+// See if there is a global application event with this name
+let auraEvt = window.$A.get("e." + eventName),
+    skuidAuraEvt = window.$A.get("e.skuid:event"),
+    paramsObject = {
+        flowName: "Civilian_Training_Form",
+        params:[{name:"cid",type:"String",value:params.row.Id}]
+    };
+
+// If there is a matching global application event with this name,
+// then fire that event, with each Skuid Event parameter
+// being mapped directly to that Event parameter.
+if (auraEvt) {
+    if (paramsObject) {
+        auraEvt.setParams(paramsObject);
+    }
+    auraEvt.fire();
+}
+
+skuidAuraEvt.setParams({
+    "name": eventName,
+    "source": "skuid",
+});
+
+// ALSO fire a generic "Skuid Event",
+// where all of the parameters are serialized to JSON
+if (paramsObject) {
+    skuidAuraEvt.setParam("data", paramsObject);
+}
+
+skuidAuraEvt.fire();</jsitem>
+		</javascript>
+		<css/>
+		<actionsequences>
+			<actionsequence id="5c35d805-abbb-4f15-bfc0-e02b9d68268d" label="test" type="event-triggered" event-scope="global" event-name="civilianTrainingFlowModalEvent">
```

Comment:
This sequence doesn't appear to be called anywhere in the example, but it also doesn't appear to do anything other than display a Toast message. Is it leftovers from debugging?